src/components/ActionBar.vue

```diff
@@ -312,7 +312,7 @@ div.bar-filter-wrapper
                     round
                     v-close-popup
                     icon="close"
-                    text-color="dark"
+                    text-color="primary"
```

Comment:
I would suggest that we make this button an `outline` type. 

![Screen Shot 2022-10-04 at 16 58 02](https://user-images.githubusercontent.com/2939354/193951573-c0fea1a8-835d-4680-9a6a-9e7796156be2.png)

This is the main page when someone is not logged in. Right now, if you look at the page for a few seconds, it's hard to identify where you should look at: login, the filtering and each active proposal.

With the voting action in mind, the login button is the most important one because it's a voting requirement. The second thing I would like people to look at is the active proposals, so they know what they could vote for. Last, in the hierarchy, I would put the filtering.

Considering this scenario, I would set the active filters as `outline` or `flat` with transparent/white background. The `or` there is because I think the primary color button for every proposal is getting more attention than the actual proposal. So, I would also change the view proposal button to the `outline` type and perhaps, then change the filtering to the `flat` style or keep them both as `outline`, whichever feels better.

Reply:
I understand completely what you mean and I agree with the premise, but unfortunately, I don't have good design skills so I really wouldn't know how to solve this from the design point of view.

I will try to follow your change indications and push a new version.

Reply:
This is how they look. Which one do you prefer to use?

The first one is using flat buttons and the second outline buttons.

![image](https://user-images.githubusercontent.com/4420760/194339568-5fd452e4-07c7-4999-99b9-940aa3760db7.png)

Reply:
I really appreciate your patience doing both and showing them side by side <3  thank you kindly, sir!

The second one gives a visual indication that the filters for status and DAO are in a different state from Sort by and Type. So yeah, the second one that uses the outline version.

Reply:
Perfect! I leave them with the outline style here [f539cc9](https://github.com/telosnetwork/app-telos-native/pull/170/commits/f539cc997edc1b303fb2c969f83011e5dd04299b)